packages/admin-client/src/common/components/CustomDatagrid/CustomDatagrid.tsx

```diff
@@ -35,7 +35,7 @@ const CustomDatagrid = <T,>({ ariaLabel, showCalculatorButton, setIsLoading, ...
     }, [identity])
 
     useEffect(() => {
-        if (setIsLoading) {
+        if (setIsLoading && isLoading != null) {
```

Comment:
Just curios can isLoading be undefined? If yes should we still perform `setIsLoading(isLoading)`?
We could make check for both null and undefined by using `!isLoading`

Reply:
`isLoading` can be undefined. If it is we shouldnt perform `setIsLoading(isLoading)`, that's correct. The problem with `!isLoading` is that it'll return true if `isLoading` is false, as `isLoading` is a boolean value. We still want to run the logic when it's false. `isLoading != null` checks for both null and undefined, so I think it's fine here.